wire-ios/Wire-iOS Tests/Utils/ZMSnapshotTestCase.swift

```diff
@@ -100,11 +51,6 @@ class ZMSnapshotTestCase: FBSnapshotTestCase {
         accentColor = .red
         snapshotBackgroundColor = UIColor.clear
 
-        // Enable when the design of the view has changed in order to update the reference snapshots
-
-        recordMode = ProcessInfo.processInfo.environment["RECORDING_SNAPSHOTS"] == "YES"
```

Comment:
Can `RECORDING_SNAPSHOTS` now be removed from the `xctestplan`s?

<img width="420" alt="Screenshot 2024-05-22 at 09 36 39" src="https://github.com/wireapp/wire-ios/assets/7196958/dcf654bc-66b5-4964-93c6-91a8b78b64f6">

Reply:
Done: https://github.com/wireapp/wire-ios/commit/ee03cf4dc7cbbf1d1a49fe9dc4312bf928aa2b9e